### PR TITLE
adding EPA spend-based (commodity) EFs

### PR DESCRIPTION
**Pull request by AnastasiaLobanova**
**Is this a new addition or an update to an existing factor?**

- [x] New
- [ ] Update

**What category or categories does it affect?**

Multiple, including the introduction of new categories

**What?**

Adding spend-based EFs from the EPA source

**Why?**

This is a part of a major addition of the spend-based EFs from various sources to the OEFDB

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -1616,3 +1616,395 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_landfill,PV
...
+Equipment,Electronics,electronics-type_printed_circuit_electronic_assembly,Printed circuit and electronic assembly,kg-CO2e/USD,0.124,,EPA,2020,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: printed circuit and electronic assembly. The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelve
+Equipment,Electronics,electronics-type_electromedical_appartuses,Electromedical appartuses,kg-CO2e/USD,0.097,,EPA,2020,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: electronic capacitors/resistors/coils/transformers/connectors and other components (except  semiconductors and printed circuit assemblies). The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelve
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv]: The description here appears to be a duplicate of the previous EF? It appears to have had a knock-on - each subsequent description matches the EF previous to it.

**Review by patstirling — CHANGES_REQUESTED** [review]
Looks super at first review - there's a couple of small changes that will need to be made to make full review possible:

- re-order by sector > category > name
- there seems to have been a mismatch in description from a certain point - marked with a comment in the DB
- `lca_activity` field: "cradle_to_shelve" -> "cradle_to_shelf"

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv]: Hm that's strange.. I will check it!

Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv]: Ok now it is resolved!

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -1679,3 +1679,397 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_landfill,PV
...
+Consumer Goods and Services,Recreation and Culture,consumer_goods-type_printing_support,Printing support,kg-CO2e/USD,0.246,,EPA,2016,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: printing support. The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
+Energy,Fuel,fuel_type_gasoline_fuels_by_products_of_petroleum_refining-fuel_use_na,Gasoline/fuels and by-products of petroleum refining,kg-CO2e/USD,0.517,,EPA,2016,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: gasoline/fuels and by-products of petroleum refining. The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
```
Comment (greenercode) [inline OpenEmissionFactorsDB.csv]: Perhaps slightly long?

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -1679,3 +1679,397 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_landfill,PV
...
+Buildings and Infrastructure,Pavement and Surfacing,pavement_surfaces-type_asphalt_shingles,Asphalt shingles,kg-CO2e/USD,0.723,,EPA,2016,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: asphalt shingles. The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
+Energy,Fuel,fuel_type_other_petroleum_coal_products-fuel_use_na,Other petroleum and coal products,kg-CO2e/USD,0.742,,EPA,2016,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: other petroleum and coal products. The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
```
Comment (greenercode) [inline OpenEmissionFactorsDB.csv]: Perhaps slightly long ID?

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -1679,3 +1679,397 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_landfill,PV
...
+Consumer Goods and Services,Recreation and Culture,consumer_goods-type_golf_courses_marinas_ski_resorts_fitness_other_rec_centers_industries,Golf courses/marinas/ski resorts/fitness and other rec centers and industries,kg-CO2e/USD,0.256,,EPA,2016,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: golf courses/marinas/ski resorts/fitness and other rec centers and industries. The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
+Restaurants and Accommodation,Accommodation,accommodation_type_hotels_campgrounds,Hotels and campgrounds,kg-CO2e/USD,0.178,,EPA,2016,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: hotels and campgrounds. The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
```
Comment (greenercode) [inline OpenEmissionFactorsDB.csv]: TBR - suggestion; accommodation-type_hotels_campgrounds

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -1679,3 +1679,397 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_landfill,PV
...
+Waste,Waste Management,waste_management-type_waste_management_remediation,Waste management and remediation,kg-CO2e/USD,0.373,,EPA,2016,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: waste management and remediation. The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
+Education,Education,education_type_elementary_secondary_schools,Elementary and secondary schools,kg-CO2e/USD,0.138,,EPA,2016,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: elementary and secondary schools. The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
```
Comment (greenercode) [inline OpenEmissionFactorsDB.csv]: lines 2036 - 2039 - ID line break. Suggestion; **education-type_elementary_secondary_schools**

File: OpenEmissionFactorsDB.csv [hunk 6]
```diff
@@ -1679,3 +1679,397 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_landfill,PV
...
+Organisational Activities,Operational Activities,organisational_activity-type_civic_social_professional_similar_organizations,Civic/social/professional and similar organizations,kg-CO2e/USD,0.137,,EPA,2016,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: civic/social/professional and similar organizations. The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
+Consumer Goods and Services,Domestic services,domestic_services-type_household_employees,Household employees,kg-CO2e/USD,0,,EPA,2016,USA,2021/10/12,Emission intensity of supply chain in US dollars spend on: household employees. The factors were prepared using USEEIO models - a life cycle models of goods and services in the US economy,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
```
Comment (greenercode) [inline OpenEmissionFactorsDB.csv]: **cradle_to_shelf** Is there a consensus on this wording for all spend-based LCA activities?

**Review by greenercode — CHANGES_REQUESTED** [review]
Other than those tiny requested changes - great PR!

**Pull request closed without merging**